zzz_generated/app-common/app-common/zap-generated/cluster-objects.h

```diff
@@ -0,0 +1,1497 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+// THIS FILE IS GENERATED BY ZAP
+
+#pragma once
+
+#include <app/data-model/DecodableList.h>
+#include <app/data-model/Decode.h>
+#include <app/data-model/Encode.h>
+#include <app/data-model/List.h>
+#include <app/util/basic-types.h>
+
+namespace chip {
+namespace app {
+namespace clusters {
+
+namespace AccountLogin {
+constexpr ClusterId kClusterId = 0x50E;
+
+} // namespace AccountLogin
+namespace AdministratorCommissioning {
+constexpr ClusterId kClusterId = 0x3C;
+// Enum for StatusCode
+enum class StatusCode : uint8_t
+{
+    STATUS_CODE_SUCCESS       = 0x00,
+    STATUS_CODE_BUSY          = 0x01,
+    STATUS_CODE_GENERAL_ERROR = 0x02,
+};
+
+} // namespace AdministratorCommissioning
+namespace ApplicationBasic {
+constexpr ClusterId kClusterId = 0x50D;
+// Enum for ApplicationBasicStatus
+enum class ApplicationBasicStatus : uint8_t
+{
+    APPLICATION_BASIC_STATUS_STOPPED                  = 0x00,
+    APPLICATION_BASIC_STATUS_ACTIVE_VISIBLE_FOCUS     = 0x01,
+    APPLICATION_BASIC_STATUS_ACTIVE_HIDDEN            = 0x02,
+    APPLICATION_BASIC_STATUS_ACTIVE_VISIBLE_NOT_FOCUS = 0x03,
+};
+
+} // namespace ApplicationBasic
+namespace ApplicationLauncher {
+constexpr ClusterId kClusterId = 0x50C;
+// Enum for ApplicationLauncherStatus
+enum class ApplicationLauncherStatus : uint8_t
+{
+    APPLICATION_LAUNCHER_STATUS_SUCCESS           = 0x00,
+    APPLICATION_LAUNCHER_STATUS_APP_NOT_AVAILABLE = 0x01,
+    APPLICATION_LAUNCHER_STATUS_SYSTEM_BUSY       = 0x02,
+};
+
+namespace ApplicationLauncherApp {
+enum FieldId
+{
+    kCatalogVendorIdFieldId = 0,
+    kApplicationIdFieldId   = 1,
+};
+
+struct Type
+{
+public:
+    uint16_t catalogVendorId;
+    Span<const char> applicationId;
+    CHIP_ERROR Encode(TLV::TLVWriter & writer, uint64_t tag) const;
```

Comment:
Would be good to visually separate the public members from the encode/decode methods – perhaps, add a 'public:' between them?￼

Reply:
just add one newline to virtually separate them, thanks